array-manipulation-2/src/includes.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+
+function includes(array, value) {
+  for (var i = 0; i <= array.length; i++) {
+    var statement = false;
+    if (array[i] === value) {
+      var statement = true;
+      break;
+    } else continue;
```

Comment:
You don't need an else continue here, it will continue if the if statements condition isnt met anyway.